src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,17 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  comments: Comment[] | null;
```

Comment:
```suggestion
  comments: Comment[]];
```
It couldn't be null now